front/components/assistant/conversation/hooks/suggestion.ts

```diff
@@ -0,0 +1,80 @@
+import { ReactRenderer } from "@tiptap/react";
+import tippy from "tippy.js";
+
+import { MentionList } from "@app/components/assistant/conversation/hooks/MentionList";
+
+export interface EditorSuggestion {
+  id: string;
+  label: string;
+  pictureUrl: string;
+}
+
+const SUGGESTION_DISPLAY_LIMIT = 7;
+
+export function makeGetAssistantSuggestions(suggestions: EditorSuggestion[]) {
```

Comment:
Typpy has limited coverage for TypeScript types, so I've used any as a temporary solution (see the PR description for details).